src/App.js

```diff
@@ -1,10 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import data from './data/dummy.json';
 
 const App = () => {
+  const [response, setResponse] = useState({});
+  const [score, setScore] = useState(0);
+  const [displayScore, setDisplayScore] = useState(false);
+
+  const handleResponse = e => {
```

Comment:
Again, `response` suggests API response, so we could rename this to something like `handleAnswerSelect`